src/theme/shared/borderRadius.ts

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+const borderRadius = {
+  xs: '2px',
+  sm: '4px',
+  lg: '8px',
+  circle: '9999px',
```

Comment:
I find this naming rather weird because it's not necessarily a circle if the width and height are not the same. Maybe we can name it for example: `max`?

Reply:
Good point, I will check with Sergiu. Thanks! 🥦

Reply:
I checked with Sergiu and he agrees with the new name key. So, `max` will be.
The change is already sync in the figma-tokens and figma. Thanks!